src/smoothscroll.js

```diff
@@ -89,11 +89,23 @@
      * @returns {Node} el
      */
     function findScrollableParent(el) {
+      var isBody;
+      var hasScrollableSpace;
+      var hasVisibleOverflow;
+
       do {
         el = el.parentNode;
-      } while (el !== d.body
-              && !(el.clientHeight < el.scrollHeight
-              || el.clientWidth < el.scrollWidth));
+
+        // set condition variables
+        isBody = el === d.body;
```

Comment:
could we bail out early if any of these conditions are true? The reason is because the `clientHeight`/`scrollHeight` properties force layout.